INSTALL.md

```diff
@@ -346,9 +346,9 @@ Directories
 
 The name of the directory under the top of the installation directory tree
 (see the `--prefix` option) where libraries will be installed.  By default
-this is "lib". Note that on Windows only static libraries (`*.lib`) will
+this is `lib/`. Note that on Windows only static libraries (`*.lib`) will
 be stored in this location. Shared libraries (`*.dll`) will always be
-installed to the "bin" directory.
+installed to the `bin/` directory.
```

Comment:
I'm not so sure about this. Even on Windows one can use `/` (as a less well-known alternative to that terrible backslash notation `\`). 
Also note that we already use the forward slash universally in many places, such as `include/openssl/configuration.h`.

Reply:
> Even on Windows one can use `/`

That totally depends.  In CMD, you cannot

Reply:
> Also note that we already use the forward slash universally in many places, such as `include/openssl/configuration.h`.

CL is one of those programs

Reply:
> That totally depends. In CMD, you cannot necessarily do so, it very much depends on the program that's being invoked.

What you cannot do with CMD is path completion involving a `/` (using the Tab key).
Yet you can do, e.g., `cd include/openssl/`.

Reply:
So even Windows users should not be too surprised about a trailing `/`, 
and if so, they can learn something new about their wonderful OS :wink:

Reply:
I was under the impression than in most non-platform-specific sections there was a deliberate intent to avoid using platform-specific things, like path separators.

But then again, as you noted we are not consistent about this and I don't think this should block this PR!

So unless @levitte feels strongly otherwise I would approve it as it is.